plugins/builds/getBuildStatuses.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const stringSchema = joi.string().regex(/^[0-9]+$/);
+const jobIdsSchema = joi.array().items(stringSchema);
```

Comment:
const schema = require('screwdriver-data-schema');
const jobIdsSchema = joi.reach(schema.models.job.base, 'id');

Reply:
parameters are from request.query so they all come in as strings